app/src/main/java/com/xfinity/blueprint_sample/mvp/view/ClickableRandomView.kt

```diff
@@ -0,0 +1,23 @@
+package com.xfinity.blueprint_sample.mvp.view
+
+import android.provider.ContactsContract.CommonDataKinds.Im
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.xfinity.blueprint_annotations.ComponentViewClass
+import com.xfinity.blueprint_annotations.ComponentViewHolder
+import com.xfinity.blueprint_sample.R
+
+
+@ComponentViewHolder(viewType = "clickable_random_view")
+class ClickableRandomViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+    var xfinity_app: ImageView = itemView.findViewById<ImageView>(R.id.imageView2)
```

Comment:
naming. Don't name variables according to some data they might hold.  Name them according to what they are for/what they do.

better names might be "icon"  and "title"